custom/airport/airportcore/resources/airportcore-items.xml

```diff
@@ -31,158 +36,272 @@
 			<value code="YELLOW" />
 		</enumtype>
 
-		<enumtype generate="true" code="daysEnum" autocreate="true"
-			dynamic="true">
-			<value code="lunedi" />
-			<value code="martedi" />
-			<value code="mercoledi" />
-			<value code="giovedi" />
-			<value code="venerdi" />
-			<value code="sabato" />
-			<value code="domenica" />
+		<enumtype generate="true" code="statePlaneEnum"
+			autocreate="true" dynamic="true">
+			<value code="assigned" />
+			<value code="maintenance" />
+			<value code="available" />
 		</enumtype>
+
+
+
+		<enumtype generate="true" code="stateFlightEnum"
+			autocreate="true" dynamic="true">
+			<value code="CANCELLED"></value>
+			<value code="ToFLIGHT"></value>
```

Comment:
naming convention per i value dell'enum è tutto in lettera grande, se ci sono più parole separarle con "_"
in questo caso TO_FLIGHT